packages/flat-lists/Lists/FlatList.js

```diff
@@ -0,0 +1,700 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+import typeof ScrollViewNativeComponent from 'react-native';
+import type {ViewStyleProp} from 'react-native/Libraries/StyleSheet/StyleSheet';
```

Comment:
Pleas can we avoid having any imports from RN internals

Reply:
Yeah. I forgot that; this way it may not work in outside projects.

Right now I'm in college, will try to change it once I reach home.

Thanks.

Reply:
Sorry for the delay. I'm busy with assignment tests at college, will try to look at this; tomorrow eod..